composer.json

```diff
@@ -14,12 +14,23 @@
     "require": {
         "php": "^7.3",
         "laravel/framework": "^7.0",
-        "pestphp/pest": "^0.1.2"
+        "pestphp/pest": "@dev"
     },
     "autoload": {
-        "files": ["src/Autoload.php"]
+        "psr-4": {
+            "Pest\\Laravel\\": "src/"
```

Comment:
Should this be `Pest\\PluginLaravel` like the other plugins, or are plugins moving to not use "Plugin" in the namespace?

If so, should the Faker and Init plugins should be moved to `Pest\Faker` and `Pest\Init` respectively?

Reply:
I was thinking about moving ( sorry ) again to `Pest\Faker`, and `Pest\Laravel`. What do you think?

Reply:
No need to be sorry, sounds good to me. :+1: I'll have a look at updating those repos tomorrow maybe?

Reply:
Thanks!